server/datastore/mysql/schema.sql

```diff
@@ -1476,6 +1497,25 @@ CREATE TABLE `software_host_counts` (
 /*!40101 SET character_set_client = @saved_cs_client */;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `software_installers` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `software_id` bigint(20) unsigned DEFAULT NULL,
+  `pre_install_query` text COLLATE utf8mb4_unicode_ci,
+  `install_script_content_id` int(10) unsigned NOT NULL,
+  `post_install_script_content_id` int(10) unsigned DEFAULT NULL,
+  `storage_id` binary(64) NOT NULL,
+  `created_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Is this intended to be used to populate the `uploaded_at` field in the API response?

Reply:
yes, should we change it to uploaded_at?